src/libs/actions/Transaction.ts

```diff
@@ -155,10 +149,8 @@ function removeWaypoint(transactionID, currentIndex) {
 /**
  * Gets the route for a set of waypoints
  * Used so we can generate a map view of the provided waypoints
- * @param {String} transactionID
- * @param {Object} waypoints
  */
-function getRoute(transactionID, waypoints) {
+function getRoute(transactionID: string, waypoints: RecentWaypoints) {
```

Comment:
Please validate this `waypoints` param, it seems to me that it should be some kind of collection, either array of record.